src/utils/normalize.js

```diff
@@ -18,6 +18,10 @@ import typeOf from 'type-of'
 
 function block(value) {
   if (Block.isBlock(value)) return value
+  if (Inline.isInline(value)) throw new Error(`Invalid \`block\` argument! It must be a block, an object, or a string. You passed: "${value}".`)
+  if (Mark.isMark(value)) throw new Error(`Invalid \`block\` argument! It must be a block, an object, or a string. You passed: "${value}".`)
+  if (Text.isText(value)) throw new Error(`Invalid \`block\` argument! It must be a block, an object, or a string. You passed: "${value}".`)
+  if (Selection.isSelection(value)) throw new Error(`Invalid \`block\` argument! It must be a block, an object, or a string. You passed: "${value}".`)
```

Comment:
Since these all throw the same error, can you refactor so that they are just a single `if` statement with OR operators?